adobe/spectrum-web-components
# feat(picker): allow customisation of popover height

## Description

Adds CSS custom property to adjust the popover height when needed.

## Related issue(s)



- https://github.com/adobe/spectrum-web-components/issues/4564

## Motivation and context

For scenarios where there are a lot of options in the dropdown and the dropdown spawns the entire screen.

## How has this been tested?




    1. Go [here](https://rocss-4564--spectrum-web-components.netlify.app/storybook/?path=/story/picker--default)
    2. Add `--sp-picker-popover-max-block-size: 100px` on the `sp-picker` component
    3. Observe the height of the popover is now 100px.

## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
